Workbooks/Workloads/Redis Insights/Redis Insights.workbook

```diff
@@ -901,7 +901,7 @@
                   "name": "SuggestedAlerts",
                   "type": 1,
                   "isRequired": true,
-                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     } ]",
+                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     },     {         \"AlertName\": \"Redis key get failures\",         \"AlertDescription\": \"Fires when key get failures exceeds threshold\",         \"AlertThreshold\": 95,         \"AlertTemplate\": \"https://azmonsolutions.blob.core.windows.net/workloadinsights/Alerts/Redis/redis-key-get-failures.json\"     } ]",

```

Comment:
Is Redis key get failures a percent? 95 seems like an odd threshold.

Reply:
Maybe should be 'Redis key get success' @austonli any thoughts?
Threshold should be 99?

Reply:
changed to 99, need to assess if should be 100